src/Test.UnitTests.Sarif/FileRegionsCacheTests.cs

```diff
@@ -595,14 +714,14 @@
                 CharOffset = 114,
                 CharLength = 600,
             };
-
+           
             var fileRegionsCache = new FileRegionsCache();
             region = fileRegionsCache.PopulateTextRegionProperties(region, uri, true, fileContent);
 
             Region multilineRegion = fileRegionsCache.ConstructMultilineContextSnippet(region, uri);
 
-            // 114 (charoffset) + 600 (charlength) + 128 (grabbing right content)
```

Comment:
> // 114 (charoffset) + 600 (charlength) + 128 (grabbing right content) [](http://example.com/codeflow?start=11&length=70)

This code comment should have been a clue to the problem! We proposed to create context regions that exceeded the size of the analysis target!

This test enforced that. :)